plugins/georeplication/gsyncd.go

```diff
@@ -11,9 +11,16 @@ import (
 )
 
 const (
-	gsyncdCommand = "/usr/local/libexec/glusterfs/gsyncd"
+	defaultGsyncdCommand = "/usr/libexec/glusterfs/gsyncd"
```

Comment:
instead of hardcoding this in code, we need to find a mechanism to fetch it from glusterfsd

example:

```"glusterfsd", "--print-gsyncdir" ```

Reply:
geo-rep is optional package(rpm) in glusterfs. Let me see if any other way to automatically extract this path. For now it can be configured `glusterd2 --gsyncd-path=<path>`

Reply:
Sent patch to glusterfs to get `libexecdir`, I will resubmit this PR once that is merged

Reply:
https://review.gluster.org/c/glusterfs/+/20759